provision.sh

```diff
@@ -177,24 +177,28 @@ if [ ! $SKIP_REQUIREMENTS ] ; then
       exit 1
   else
     # Install virtualenv
-    which -a virtualenv >> /dev/null
+    which -a pipenv >> /dev/null
     if [[ $? != 0 ]] ; then
-      pip install virtualenv
+      pip install pipenv
     fi
+    cd $ROOT/ansible
+    VENV=`pipenv --venv`
+
     # Create a virtualenv for this project and use it for ansible
-    if [ ! -f $ROOT/.virtualenv ]; then
-      virtualenv --python=python2.7 $ROOT/ansible/.virtualenv
+    if [ ! $VENV ]; then
```

Comment:
@tharna I don't think this is needed. If one runs `pipenv install` it automatically checks for a virtual environment and creates one if it's missing.

Reply:
True, it does create the env if there isn't one. Removed.